packages/ui/src/components/atoms/Container.tsx

```diff
@@ -29,3 +38,14 @@ Container.defaultProps = {
   size: 'lg',
   $noFlex: false,
 };
+
+export const CustomContainer = styled(Container)`
+  flex-direction: column;
+  gap: 32px;
+  z-index: 1;
+`;
```

Comment:
Can we rename this to something specific?

Reply:
done